src/helpers/shutter.ts

```diff
@@ -107,7 +107,7 @@ export async function setProposalKey(params) {
     log.info(`[shutter] proposal scores updated for ${proposalId}`);
   } catch (e) {
     capture(e);
-    log.error(`[shutter] setProposalKey failed ${JSON.stringify(e)}`);
```

Comment:
Maybe we can leave it an error. Slightly more important

Reply:
In that case, we should also move all `log.warn` to `log.error`, unless they're validation error (in ingestor).

Reply:
Let's not forget we also capture the error on Sentry, to that log should be gone